packages/table/src/Table.tsx

```diff
@@ -1,19 +1,84 @@
 import React from 'react';
+import debounce from 'lodash/debounce';
+import { transparentize } from 'polished';
 import { cx, css } from '@leafygreen-ui/emotion';
 import { uiColors } from '@leafygreen-ui/palette';
+import {
+  useIsomorphicLayoutEffect,
+  useViewportSize,
+} from '@leafygreen-ui/hooks';
 import { HeaderRowProps } from './HeaderRow';
 import { TableHeaderProps } from './TableHeader';
 import { TableProvider } from './TableContext';
 import TableHead from './TableHead';
 import TableBody from './TableBody';
 import { SortProvider } from './SortContext';
 
+const shadowColor = transparentize(0.6, uiColors.black);
+
+const containerStyle = css`
+  position: relative;
+  max-width: calc(100% - 100px);
+`;
+
 const tableStyles = css`
   border-collapse: collapse;
   box-sizing: border-box;
   border-bottom: 1px solid ${uiColors.gray.light2};
 `;
 
+const shadow = css`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  width: 20px;
+  overflow: hidden;
+
+  &:after {
+    opacity: 0;
+    content: '';
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    width: 6px;
```

Comment:
Right now, the shadow doesn't form a hard edge on the outer side of the shadow. My suggestion:

- Bump this width value to at least `12px`.
- Change the x offset of the shadows to `4px` and `-4px`.
- Reduce Alpha value of the box shadow RGBA from `0.4` to `0.3`.

Doing these things will make this look much more polished.

Reply:
👍